bcda/bcdacli/cli_test.go

```diff
@@ -270,7 +277,6 @@ func (s *CLITestSuite) TestArchiveExpiring() {
 	defer f.Close()
 
 	// execute the test case from CLI
-	os.Setenv("ARCHIVE_THRESHOLD_HR", "0")
```

Comment:
Since other tests (like api_test.go) are manually creating jobs, we can encounter a situation where this archive-job-files command is run on invalid job metadata.

By keeping the ARCHIVE_THRESHOLD_HR the same (24h), we won't pick up these invalid jobs to clean up.